doc/nrf/libraries/caf/sensor_data_aggregator.rst

```diff
@@ -0,0 +1,73 @@
+.. _caf_sensor_data_aggregator:
+
+CAF: Sensor data aggregator module
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |sensor_data_aggregator| of the :ref:`lib_caf` (CAF) is a simple, module responsible for aggregating sensor data in form of sensor_event and passing them further in packages.
+
+Configuration
+*************
+
+The following Kconfig options are required:
+
+* :kconfig:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR` - This option enables the |sensor_data_aggregator|.
+
+To use the module, you must complete the following steps:
+
+1. Enable the :kconfig:`CONFIG_CAF_SENSOR_DATA_AGGREGATOR` option.
+#. Add the app.overlay file that describes the aggregator parameters.
+   Each aggregator should be placed as a separate node.
+   For example, the file content could look like follows:
+
+   .. code-block:: devicetree
+
+      agg0: agg0 {
+              compatible = "caf,aggregator";
+              sensor_descr = "accel_sim_xyz";
+              buf_data_length = <120>;
+              sensor_data_size = <12>;
+              status = "okay";
+      };
+
+      agg1: agg1 {
+             compatible = "caf,aggregator";
+             sensor_descr = "void_test_sensor";
+             buf_data_length = <80>;
+             sensor_data_size = <8>;
+             status = "okay";
+      };
+
+Here there are defined two aggregators each of which is responsible for handling data of different sensor.
```

Comment:
`In this example two aggregators are defined, that aggregates data from different sensors`.